parquet-thrift/src/main/java/org/apache/parquet/thrift/struct/ThriftType.java

```diff
@@ -37,19 +37,14 @@
 import java.util.List;
 import java.util.Map;
 
-import org.codehaus.jackson.annotate.JsonCreator;
-import org.codehaus.jackson.annotate.JsonIgnore;
-import org.codehaus.jackson.annotate.JsonProperty;
-import org.codehaus.jackson.annotate.JsonSubTypes;
-import org.codehaus.jackson.annotate.JsonTypeInfo;
-import org.codehaus.jackson.annotate.JsonTypeInfo.As;
-import org.codehaus.jackson.annotate.JsonTypeInfo.Id;
+import com.fasterxml.jackson.annotation.*;
```

Comment:
I was just about to merge this when I noticed the wildcard import. In Parquet we aim to avoid that. Could you please add each imported class separately? Thanks.

Reply:
Very good point @zivanfi, my IDE collapsed the import into a wildcard. I prefer separate explicit imports as well.